testplan/web_ui/testing/src/Nav/Nav.js

```diff
@@ -18,6 +18,11 @@ const Nav = (props) => {
     props.selected,
   );
 
+  let selectedUid = undefined;
```

Comment:
slightly prefer `null` to `undefined` here. `undefined` means not defined (usually a mistyped variable name), `null` means defined but empty.